### ISSUE TITLE
Allow to browse provided pdb files

### ISSUE THREAD

**Issue by GiovanniBussi** [plumed-nest/plumed-nest#38]
(This was suggested by @mbernett).

We could improve the syntax highlighter to understand when a command points to a pdb file and upload also the pdb file so that users can see it by clicking on it (similarly to the way now one can see included files).

**Comment by maxbonomi** [comment]
Should we impose a file size limit for the pdb?
Just not to make our rep explode

Max

**Comment by GiovanniBussi** [comment]
That's true (and in principle true also for input file and, most likely, for stdout files). I will check if we can store them gzipped. And we can additionally put a cap on the size of individual files to be stored on the server.

**Comment by maxbonomi** [comment]
Great!

**Comment by GiovanniBussi** [comment]
changed `browse` to `download` in the issue name. Actually we might make available the gzipped pdb files.
